Orange/widgets/gui.py

```diff
@@ -2049,7 +2049,10 @@ def item2name(self, item):
 
     def __setitem__(self, index, item):
         def unselect(i):
-            item = self.listBox.item(i)
+            try:
+                item = self.listBox.item(i)
+            except RunTimeError:  # Underlying C/C++ object has been deleted
```

Comment:
This is probably hard to test, since we don't know how to reproduce the error, right?
I am guessing `RunTimeError` is a typo of `RuntimeError`.